### ISSUE TITLE
Change Burger Icon

### ISSUE THREAD

**Issue by rohanisme** [yarolegovich/SlidingRootNav#69]
After changing the burger icon, on toggling the icon remain same its not changing to back arrow how can i change it.
Please help me!

**Comment by uzairmughal3397** [comment]
did you find something to change the icon?

**Comment by rohanisme** [comment]
Yes, i did find a way instead of using the predefined icons i set a linear layout with custom icons and custom on clicklisteners, so on click slidenav.close or open was defined based on the operation required. it worked like a charm
